__tests__/commands/dev-env-sync-sql.js

```diff
@@ -93,6 +105,18 @@ describe( 'commands/DevEnvSyncSQLCommand', () => {
 
 			expect( cmd.searchReplaceMap ).toEqual( { 'test.go-vip.com': 'test-slug.vipdev.lndo.site' } );
 		} );
+
+		it( 'should return a map of search-replace values for multisite', () => {
+			const cmd = new DevEnvSyncSQLCommand( app, msEnv, 'test-slug' );
+			cmd.slug = 'test-slug';
+			cmd.siteUrls = [ 'test.go-vip.com', 'subsite.com' ];
```

Comment:
Should we add tests for subsites that doesn't have a domain?

Reply:
Would there ever be a case for subsite to not have domain? The `wp_blogs` [table's](https://codex.wordpress.org/Database_Description#Table:_wp_blogs) `domain` column is not nullable so I'm not sure if that will ever occur 🤔

Reply:
[Here](https://wordpress.org/documentation/article/before-you-create-a-network/#path-based)

Thinking back though, they could just go to `test-slug.vipdev.lngdo.site/subsite` to access it in that case, actually. No action needed then!